package.json

```diff
@@ -1,41 +1,46 @@
-{
-  "name": "yggdrasil",
-  "version": "1.5.1",
-  "author": "Zeke Sonxx <zeke@zekesonxx.com>",
-  "description": "Mojang authentication (Yggdrasil) client",
-  "scripts": {
-    "test": "mocha -R spec",
-    "pretest": "npm run lint && npm run build",
-    "prepublish": "npm run lint && npm run build",
-    "lint": "ts-standard",
-    "fix": "ts-standard --fix",
-    "build": "npx tsc"
-  },
-  "main": "./lib/index",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/PrismarineJS/node-yggdrasil.git"
-  },
-  "bugs": {
-    "url": "https://github.com/PrismarineJS/node-yggdrasil/issues"
-  },
-  "keywords": [
-    "minecraft",
-    "yggdrasil",
-    "authentication"
-  ],
-  "dependencies": {
-    "node-fetch": "^2.6.1",
-    "uuid": "^8.2.0"
-  },
-  "license": "MIT",
-  "devDependencies": {
-    "@types/node": "^14.14.19",
-    "@types/node-fetch": "^2.5.8",
-    "@types/uuid": "^8.3.0",
-    "mocha": "^7.2.0",
-    "nock": "^13.0.2",
-    "ts-standard": "^10.0.0",
-    "typescript": "^4.1.3"
-  }
-}
+{
+  "name": "yggdrasil",
+  "version": "1.5.2",
+  "author": "Zeke Sonxx <zeke@zekesonxx.com>",
+  "description": "Mojang authentication (Yggdrasil) client",
+  "files": [
+    "lib/**/*"
```

Comment:
Let's not use that, just un ignore these files by creating a .npmignore file that doesn't contain lib

Reply:
will do, could you explain why?

Reply:
It's better to publish all the files (unless they are very big) so users can see what they use in node_modules folder